examples/sdk-examples/src/examples/internationalDateFormat/dateFilter/index.tsx

```diff
@@ -0,0 +1,73 @@
+// (C) 2007-2019 GoodData Corporation
+/* eslint-disable import/no-unresolved,import/default */
+import React from "react";
+
+import { ExampleWithSource } from "../../../components/ExampleWithSource";
+
+import { DateFilterComponentExample_DDMMYYYY } from "./DateFilterComponentExample_DDMMYYYY";
+import DateFilterComponentExample_DDMMYYYYSRC from "!raw-loader!./DateFilterComponentExample_DDMMYYYY";
+import DateFilterComponentExample_DDMMYYYYSRCJS from "!raw-loader!../../../../examplesJS/internationalDateFormat/dateFilter/DateFilterComponentExample_DDMMYYYY";
+
+import { DateFilterComponentExample_MDYY } from "./DateFilterComponentExample_MDYY";
+import DateFilterComponentExample_MDYYSRC from "!raw-loader!./DateFilterComponentExample_MDYY";
+import DateFilterComponentExample_MDYYSRCJS from "!raw-loader!../../../../examplesJS/internationalDateFormat/dateFilter/DateFilterComponentExample_MDYY";
+
+import { DateFilterWithColumnChartExample_DDMMYYYY } from "./DateFilterWithColumnChartExample_DDMMYYYY";
+import DateFilterWithColumnChartExample_DDMMYYYYSRC from "!raw-loader!./DateFilterWithColumnChartExample_DDMMYYYY";
+import DateFilterWithColumnChartExample_DDMMYYYYSRCJS from "!raw-loader!../../../../examplesJS/internationalDateFormat/dateFilter/DateFilterWithColumnChartExample_DDMMYYYY";
+
+import { DateFilterWithColumnChartExample_MDYY } from "./DateFilterWithColumnChartExample_MDYY";
+import DateFilterWithColumnChartExample_MDYYSRC from "!raw-loader!./DateFilterWithColumnChartExample_MDYY";
+import DateFilterWithColumnChartExample_MDYYSRCJS from "!raw-loader!../../../../examplesJS/internationalDateFormat/dateFilter/DateFilterWithColumnChartExample_MDYY";
+
+export const InternationalDateFilterExample: React.FC = () => (
+    <div>
+        <h1>Date Filter</h1>
+
+        <p>
+            These examples show how to pass dateFormat property to a DateFilter component for dates to be
+            displayed in the desired format. For more details about date formats, please see{" "}
+            <a href="https://date-fns.org/docs/Getting-Started">here</a>.
```

Comment:
Please use https://date-fns.org/docs/format so that it goes to the relevant page. Also, please open the link in a new tab.